### ISSUE TITLE
Allow multiple errors

### ISSUE THREAD

**Issue by ndw** [xproc/3.0-test-suite#51]
The current test suite allows a single error code, for example, `ab-p-document015.xml`:
```xml
<t:test xmlns:t='http://xproc.org/ns/testsuite/3.0'
        xmlns:err="http://www.w3.org/ns/xproc-error"
        expected="fail" code="err:XD0023">
    <t:title>p:document 015</t:title>
    ...
```
I'd like to change the schema so that multiple error codes are allowed. In particular, I get a file not found error for the missing dtd in this test. I think that's a perfectly reasonable response code, but I don't think "invalid" is unreasonable.
In general, it's hard to get completely identical errors across implementations. In the 1.0 test suite, we let any error succeed. I think we can try to do a little better here.

**Comment by xml-project** [comment]
I agree. 
What about changing "@code" to a whitespace separated list of error codes and say, that the test is successful, if at least one of the listed error codes is matched?

**Comment by ndw** [comment]
Exactly what I had in mind. I'm working on it now.
I'm also adding some more metadata to the test-suite schema and I'll arrange it so that the test suite is validated on each PR.

**Comment by xml-project** [comment]
👍

**Comment by xml-project** [comment]
Solved with pr #52
